src/FluentCMS.Application/Sites/AddUrlSiteCommand.cs

```diff
@@ -0,0 +1,9 @@
+﻿using MediatR;
+
+namespace FluentCMS.Application.Sites;
+
+public class AddUrlSiteCommand : IRequest
```

Comment:
I think these names are better for this command:
- AddUrlToSiteCommand
- AddSiteUrlCommand

I prefer `AddSiteUrlCommand`